rental.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'book'
+require_relative 'person'
+
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    book.add_rental << self
+    person.add_rental << self
```

Comment:
:red_circle: I think add_rental is a function that takes a rental instance as a value but here you are using it as an array. What do you think if you just push to the rentals array that belongs to the object? Here is our you can alter your code :arrow_down: 
```

 book.rentals << self
    person.rentals << self
```
# or 

```
book.rentals.push(self)
person.rentals.push(self)
```